Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# 2019-09 -- v4.1.5
+# 2019-10-22 -- v4.1.6
```

Comment:
Looks like versions are off. The release spreadsheet shows 4.1.7.

Reply:
Updated the missing version.